### ISSUE TITLE
protein_graph_train, _ = load_graphs('data/' + dataset + '/processed' + '/pocket_graph/' + str(id) + '.bin')

### ISSUE THREAD

**Issue by xinxinxin12366666** [JK-Liu7/AttentionMGT-DTA#7]
麻烦请问下这个pocket_graph数据从哪里获得呀

**Comment by divyavani01** [comment]
是的，连我也有同样的疑问

**Comment by LACAD** [comment]
These files are created when you execute the protein_process.py script.

**Comment by Ppll123** [comment]
> 这些文件是在执行protein_process.py脚本时创建的。

请问你有成功运行改代码吗？

**Comment by angeload** [comment]
> 请问你有成功运行改代码吗？

Yes, but there is another problem with the code. The train_DTA.py has an errror. I have asked the author about how to fix it but he never gave me an answer. You can see my previous post about it.

**Comment by JK-Liu7** [comment]
Thank you for your interest in our work. The generated ‘pocket_graph’ file has been uploaded.

**Comment by divyavani-d** [comment]
i was unable to get ESM_embeddings.npy files for protein sequence ....though i tried its showing error....how did u guys extract

**Comment by angeload** [comment]
> i was unable to get ESM_embeddings.npy files for protein sequence ....though i tried its showing error....how did u guys extract

You have to generate them. Use the ESM repository https://github.com/facebookresearch/esm.
